src/core/vector/qgsvectorlayerfeaturecounter.cpp

```diff
@@ -91,16 +97,26 @@ bool QgsVectorLayerFeatureCounter::run()
       if ( isCanceled() )
       {
         mRenderer->stopRender( renderContext );
+        mFeedback.reset();
```

Comment:
does this need the feedback reset on the expression context too here?

Reply:
Good catch, yes